docs/root/configuration/overview/extension.rst

```diff
@@ -90,3 +92,20 @@ common subscription statistics, it also provides the following:
   config_reload, Counter, Total number of successful configuration updates
   config_fail, Counter, Total number of failed configuration updates
   config_conflict, Counter, Total number of conflicting applications of configuration updates; this may happen when a new listener cannot reuse a subscribed extension configuration due to an invalid type URL.
+
+
+Listener Filters
+
+For Listener filters, the discovery service configuration is: :ref:`dynamic listener filter
+re-configuration<envoy_v3_api_field_config.listener.v3.ListenerFilter.config_discovery>`.
+Note, the dynamic listener filter config is only supported in TCP listeners.
```

Comment:
Do we plan to release the TCP first? then support UDP in the future. If yes, what happened if I set the ECDS for the UDP, do we have an explicit error message to tell the user the UDP doesn't support the ECDS yet? I thought it will be better to not silently ignore ECDS for UDP and an explicit message for the user.

Reply:
Yeah, that's what I am thinking.
Just added a check and a RELEASE_ASSERT to crash Envoy for UDP listeners with ECDS listener filter configured.

Reply:
emm... I prefer to raise normal exception, not a crash, but we can see other's opinion

Reply:
Exception in the config-path digestion code makes sense, as Envoy will reject any config that is incorrect.